cmd/bb_worker/main.go

```diff
@@ -150,12 +150,22 @@ func main() {
 			if err != nil {
 				log.Fatal("Failed to create eviction set for cache directory: ", err)
 			}
+			var linkHelper cas.LinkHelper
```

Comment:
What are your thoughts on changing the naming to something like this?
- HardlinkingFileFetcher -> CachingFileFetcher
- LinkHelper -> FileInstaller
  - LinkHelperHardlink -> HardlinkingFileInstaller
  - LinkHelperClonefile -> CloningFileInstaller

I don't think HardlinkingFileFetcher is accurate anymore, now that it can also be used to clone files.

Reply:
Installer doesn't ring true to me, but I don't know what else does.  To me it's more like we are "building" the input/output tree from the cache (or vice versa), but don't know that Builder is any more intuitive.  Ultimately you will probably look at this code a lot more than I will, so I'm easy if you're confident in Installer.

Reply:
FileInstaller, FilePlacer, FilePopulator, any will do. The point being: LinkHelper is inaccurate here, because it doesn't link files.

My suggestion was also to let "Cloning" and "Hardlinking" be prefixes instead of suffixes, as it's more consistent with the rest of the codebase.